CMakeLists.txt

```diff
@@ -195,47 +181,40 @@ if(Launcher_LAYOUT_REAL STREQUAL "mac-bundle")
     # Add the icon
     install(FILES ${Launcher_Branding_ICNS} DESTINATION ${RESOURCES_DEST_DIR} RENAME ${Launcher_Name}.icns)
 
-elseif(Launcher_LAYOUT_REAL STREQUAL "lin-nodeps")
+elseif(UNIX)
     set(BINARY_DEST_DIR "bin")
-    set(LIBRARY_DEST_DIR "bin")
-    set(PLUGIN_DEST_DIR "plugins")
-    set(BUNDLE_DEST_DIR ".")
-    set(RESOURCES_DEST_DIR ".")
-    set(JARS_DEST_DIR "bin/jars")
-
-    # install as bundle with no dependencies included
-    set(INSTALL_BUNDLE "nodeps")
-
-    # Set RPATH
-    SET(Launcher_BINARY_RPATH "$ORIGIN/")
-
-    # Install basic runner script
-    configure_file(launcher/Launcher.in "${CMAKE_CURRENT_BINARY_DIR}/LauncherScript" @ONLY)
-    install(PROGRAMS "${CMAKE_CURRENT_BINARY_DIR}/LauncherScript" DESTINATION ${BUNDLE_DEST_DIR} RENAME ${Launcher_Name})
+    if(Launcher_PORTABLE)
+        set(LIBRARY_DEST_DIR "bin")
+        set(BUNDLE_DEST_DIR ".")
+        set(JARS_DEST_DIR "bin/jars")
 
-elseif(Launcher_LAYOUT_REAL STREQUAL "lin-system")
-    set(Launcher_BINARY_DEST_DIR "bin" CACHE STRING "Path to the binary directory")
-    set(Launcher_LIBRARY_DEST_DIR "lib${LIB_SUFFIX}" CACHE STRING "Path to the library directory")
-    set(Launcher_SHARE_DEST_DIR "share/polymc" CACHE STRING "Path to the shared data directory")
-    set(JARS_DEST_DIR "${Launcher_SHARE_DEST_DIR}/jars")
-    set(Launcher_DESKTOP_DEST_DIR "share/applications" CACHE STRING "Path to the desktop file directory")
-    set(Launcher_METAINFO_DEST_DIR "share/metainfo" CACHE STRING "Path to the metainfo directory")
-    set(Launcher_ICON_DEST_DIR "share/icons/hicolor/scalable/apps" CACHE STRING "Path to the scalable icon directory")
+        # launcher/Application.cpp will use this value
+        set(Launcher_APP_BINARY_DEFS "-DLauncher_PORTABLE=1")
```

Comment:
```suggestion
        set(Launcher_APP_BINARY_DEFS "-DLauncher_PORTABLE")
```

Reply:
other settings use DLauncher

Reply:
@dada513 You are mistaking macro definitions with CMake option definitions. The macros use `LAUNCHER_` and the CMake options use `Launcher_`.

Reply:
@LennyMcLennington `Application.cpp` needs `Launcher_PORTABLE=1` or it doesn't work. I couldn't find a better way for it to check the boolean. Did you see if the logic works well after changing it?

Also, we decided to change the flags to `POLYMC_*` in a future PR, which is why I left it lowercase to fit in with the other flags.

Reply:
> @LennyMcLennington `Application.cpp` needs `Launcher_PORTABLE=1` or it doesn't work. I couldn't find a better way for it to check the boolean. Did you see if the logic works well after changing it?

I know C and C++ so I am pretty confident that it will work. Read the change I suggested in Application.cpp.

> Also, we decided to change the flags to `POLYMC_*` in a future PR, which is why I left it lowercase to fit in with the other flags.

Changing the CMake flags to `PolyMC_` has literally nothing to do with naming conventions for C Preprocessor macros.